lang/cpp.md

```diff
@@ -0,0 +1,16 @@
+---
+lang: C++
+syntax: C++
+title: C++ is amazing!
+author: motasimmakki
+---
+
+```c++
+#include<iostream>
+int main() {
+    std::cout<< "Hello, World";
+    return 0;
+}
+\```

```

Comment:
Remove the `\` here. 🙂

Reply:
> Also, I think this is a duplicate of #41.

I did not notice that, and #41 is also not merged yet, I think.

Reply:
@motasimmakki: But the work has already done by @Panquesito7, also he's the moderator of [**DEVSTRONS'**](https://discord.gg/MVujzTBqed) :sunglasses: 

@vsnthdev: ig, we've to consider this spam request, & need to close this!

Reply:
Okay No problem, happy learning to me!🙃